src/index.html

```diff
@@ -28,14 +29,8 @@
       </ul>
     </div>
   </nav>
-<<<<<<< HEAD
-  <section class="container d-flex my-4 py-3 flex-wrap justify-content-evenly g-4  border border-dark border-3">
-    <div class="row g-3 justify-content-evenly" id="board">
-    </div>
-=======
   <section class="container d-flex my-4 py-3 flex-wrap justify-content-evenly g-4">
     <div class="row g-3 justify-content-between" id="board"></div>
->>>>>>> dev
```

Comment:
Let's address this conflict and keep either `justify-content-evenly `or use `justify-content-start`. With `between`, it seems to me like there is an item missing in the center.
![image](https://user-images.githubusercontent.com/59420634/143496865-3b15e1c9-a906-4f5e-8098-620c268b06de.png)

Reply:
Fixed